language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/context/OrderByClauseNodeContext.java

```diff
@@ -93,18 +94,21 @@ public void sort(BallerinaCompletionContext context,
                 TypeDescKind.BOOLEAN, TypeDescKind.FLOAT,
                 TypeDescKind.DECIMAL);
 
+        QueryExpressionNode queryExprNode = (QueryExpressionNode) node.parent().parent();
```

Comment:
Can we always guarantee that the node.parent().parent()  is a QueryExpressoin node? IMO, this is an unsafe cast. Shall we try a loop where we traverse the parents until we find the QueryExpression node kind? WDYT @gayalkuruppu